app/views/users/_groups.html.erb

```diff
@@ -27,14 +27,12 @@ See docs/COPYRIGHT.rdoc for more details.
 
 ++#%>
 <%= labelled_tabular_form_for(:user, url: { action: 'update' }, html: {method: :put}) do %>
+  <%= link_to  t(:label_manage_groups), groups_path, class: 'button -highlight' %>
   <section class="form--section">
     <% Group.all.sort.each do |group| %>
-      <label class="form--label-with-check-box">
-        <%= styled_check_box_tag 'user[group_ids][]', group.id, @user.groups.include?(group) %>
-        <%= group %>
-      </label>
+      <ul>
+        <li><%= link_to group.name, group_path(group) %></li>
+      </ul>
     <% end %>
-    <%= hidden_field_tag 'user[group_ids][]', '' %>
   </section>
-  <%= styled_button_tag l(:button_save), class: '-highlight -with-icon icon-checkmark' %>
```

Comment:
The view is now confusing to me. In this: 
![image](https://user-images.githubusercontent.com/617519/81069167-d2953100-8ee1-11ea-9692-68a235e2813d.png)
it looks as if the user where a member of the group which he is not.

I am fine with removing this functionality but IMO we should list only the active groups here (and have a message in case the user isn't member in any group) and then have the button to manage memberships at the bottom of that list.
Or we have a list of all groups with an indication of whether the user is a member of that list and an edit link (e.g. pencil icon) behind each that links to `/admin/groups/:id/edit?tab=users`.

Reply:
I absolutely missed that one, of course only active groups should be added. I removed the functionality here since you may no longer push into the association directly without calling the service. We could modify the request to have a select to add the user to the group and then `POST /groups/:group_id` with the user in question.

I would vote to remove it for now, only showing the active groups (with an edit icon) and see if this bugs anyone. I would think no one uses this view anyway.

Reply:
Only showing the active groups is fine with me.